android/src/toga_android/widgets/textinput.py

```diff
@@ -24,6 +24,18 @@ def onTextChanged(self, _charSequence, _start, _before, _count):
         pass
 
 
+class TogaReturnListener(OnKeyListener):
+    def __init__(self, impl):
+        super().__init__()
+        self.impl = impl
+        self.interface = impl.interface
+
+    def onKey(self, _view, _key, _event):
+        if (int(_key) == 66 or int(_key) == 160) and int(_event.getAction()) == 1:
```

Comment:
Is there a system constant for these "magic numbers"? If not... could we add some? We have `toga_gtk/keys.oy` et al; this doesn't currently exist for Android, but will probably need to.

Reply:
There are constants for them but after looking at all the constants for android, not all of the keyboard keys actually have their own constants. ! and ? for example on the android keyboard are number keys with a shift mod so a similar file to keys.py would be useful if key detection is wanted more in the future. I am starting work on getting that done.

Should the keys.py file for android be it's own PR since I also had to edit the toga_gtk/keys.py to add the numpad key to the file or could they both be submitted with this one.

Reply:
A new keys.py for Android doesn't need to be its own PR. If this PR needs a keys reference, it's fine to include in this PR.